reference/dic_tags.rst

```diff
@@ -1175,6 +1176,55 @@ also register it manually:
     The ``priority`` value is optional and defaults to ``0``.
     The higher priority loaders are tried first.
 
+.. _reference-dic-tags-twig-runtime:
+
+twig.runtime
+--------------
+
+**Purpose**: To register a custom Lazy-Loaded Twig Extension
+
+To enable a Lazy Loaded Twig extension, add it as a regular service in one of your
+configuration and tag it with ``twig.runtime``. If you're using the
+:ref:`default services.yaml configuration <service-container-services-load-example>`,
+the service is auto-registered and auto-tagged. But, you can also register it manually:
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        services:
+            App\Twig\AppExtension:
+                tags: [twig.runtime]
+
+    .. code-block:: xml
+
+        <?xml version="1.0" encoding="UTF-8" ?>
+        <container xmlns="http://symfony.com/schema/dic/services"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xsi:schemaLocation="http://symfony.com/schema/dic/services
+                http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+            <services>
+                <service id="App\Twig\AppExtension">
+                    <tag name="twig.runtime" />
+                </service>
+            </services>
+        </container>
+
+    .. code-block:: php
+
+        use App\Twig\AppExtension;
+        use App\Twig\AnotherExtension;
+
+        $container
+            ->register(AppExtension::class)
+            ->addTag('twig.runtime')
+        ;
+       
+For information on how to create the actual Runtime Twig Extension class, see
+`Twig's documentation`_ on the topic or read the
+:doc:`/templating/twig_extension#creating-lazy-loaded-twig-extensions` article.
```

Comment:
If we want to link to a particular section, we will have to use the `ref` role like this:

```rst
:ref:`Creating Lazy-Loaded Twig Extensions <lazy-loaded-twig-extensions>`
```

and the headline in `templating/twig_extension.rst` must be prefix now with the desired label:

```rst
.. _lazy-loaded-twig-extensions:

Creating Lazy-Loaded Twig Extensions
------------------------------------
```